backend/app/Http/Actions/Diary/ShowSingleDiaryAction.php

```diff
@@ -25,7 +25,8 @@ public function __construct(
 
     public function __invoke($uuid): View|Factory|Redirector|RedirectResponse
     {
-        $diary = Diary::where("uuid", $uuid)->first();
+        dd(Diary::where("uuid", $uuid)->first()->statistic_per_individual, Diary::with('statistic_per_individual')->where("uuid", $uuid)->first());
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Relation 'statistic_per_individual' is not found in App\Models\Diary model.